redfish-client/redfish-client

```diff
@@ -210,6 +214,20 @@ if __name__ == '__main__':
     '''Main application redfish-client'''
     # Functions
 
+    def special_location():
+        '''Check if the installation is in a special location
+        (virtual env, home, etc...)
+
+        :returns: prefix or False
+        :rtype: string or bool
+        '''
+        if(distutils.sysconfig.PREFIX != '/usr' and
+           distutils.sysconfig.PREFIX != '/usr/local'):
+            return(distutils.sysconfig.PREFIX)
+        elif(os.getenv('VIRTUAL_ENV')):
+            return(os.getenv('VIRTUAL_ENV'))
+        return(False)
+
```

Comment:
The fact that there are multiple places I agree. The fact that we handle it in the code, instead of externally to the code is our choice. And as you understood, I'm advocating for the second ;-)